package.json

```diff
@@ -22,6 +22,8 @@
   },
   "homepage": "https://github.com/metro-ontime/frontend#readme",
   "dependencies": {
+    "@date-io/core": "^1.2.0",
```

Comment:
material-ui/pickers already supports moment-js according to this page: https://material-ui-pickers.dev/getting-started/installation
Can we use @date-io/moment ?